pkg/shell/hosts_dialog.jsx

```diff
@@ -495,7 +595,6 @@ class HostKey extends React.Component {
                     <div>{_("The fingerprint should match:")} {fingerprint_help}</div>
                     <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-verify-help hostkey-fingerprint pf-v5-u-font-family-monospace">{fp}</ClipboardCopy>
                 </ExpandableSection>
-                <Alert variant='warning' isInline isPlain title={_("Malicious pages on a remote machine may affect other connected hosts")} />
```

Comment:
Was this dropped intentionally? I thought we wanted to stay bug-for-bug compatible in this PR, and I don't see it anywhere else.

Reply:
No, by mistake. Thanks!

Reply:
Fixed.